src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/ServiceJourneys/Editor/PassingTimesEditor/index.js

```diff
@@ -138,15 +138,15 @@ class PassingTimesEditor extends Component {
           <div className="stop-place">Stoppested</div>
           <div className="table-headers">
             <div className="collection">
-              <div className="upper">Ankomst</div>
+              <div className="upper">Avgang</div>
```

Comment:
Don't you also need to switch the children (normal, senest vs tidligst normalt) ?

Reply:
Godt observert!

Reply:
(fixed)